### DISCUSSION TITLE
Waybar zoom

### DISCUSSION THREAD

**Discussion by abenezerw** [prasanthrangan/hyprdots#739] [General]
Should this be zoomed in like that or?
![image](https://github.com/prasanthrangan/hyprdots/assets/46224293/ed87fdef-bf15-4d2b-8aff-a57715eddfef)
The other styles are normal size, but this one(above) in particular is big. 
![image](https://github.com/prasanthrangan/hyprdots/assets/46224293/fee3ffa0-9e00-49f0-956f-97b31730c52b)

**Comment by kRHYME7** [comment]
https://github.com/prasanthrangan/hyprdots/wiki/Theming#waybar-config

Here you go.
